packages/fhir-team-management/src/constants.tsx

```diff
@@ -0,0 +1,26 @@
+// url
+export const URL_ADMIN = 'admin';
+export const URL_LIST_TEAMS = `${URL_ADMIN}/teams/`;
+export const URL_ADD_TEAM = 'teams/add/';
+export const URL_EDIT_TEAM = 'teams/edit/';
+
+//
+// Practitioner
+//
+export const PRACTITIONERROLE_GET = 'PractitionerRole/';
+export const PRACTITIONERROLE_POST = 'PractitionerRole/';
+export const PRACTITIONERROLE_DEL = 'PractitionerRole/';
```

Comment:
nitpick: looks like these should be `PRACTITIONER_ROLE`_*`

Reply:
i made them separate myself so that we can adjust them in future if required